specs/eip4844/polynomial-commitments.md

```diff
@@ -187,6 +187,10 @@ def hash_to_bls_field(polys: Sequence[Polynomial],
     for commitment in comms:
         data += commitment
 
+    # Append challenge index last so we can cache the data buffer
+    # in the case of multiple challenges
+    data += int.to_bytes(challenge_index, 1, ENDIANNESS)
```

Comment:
This hashing is done inside of the cryptography libraries, nevertheless, I like this change because it means that the cryptography code will not need to clone and I think it's more performant as the blob sizes grow